packages/nextjs/src/serverComponentClient.ts

```diff
@@ -34,9 +34,7 @@ class NextServerComponentAuthStorageAdapter extends CookieAuthStorageAdapter {
 	}
 }
 
-export const createRouteHandlerSupabaseClient = createServerComponentSupabaseClient;
```

Comment:
I think this one was accidentally left in from previous PR

Reply:
Yeah I think so.